src/middlewares/validations/bookAccomodation.js

```diff
@@ -1,34 +1,34 @@
 import Joi from '@hapi/joi';
 
 export default (req, res, next) => {
-    const schema = Joi.object({
-        accommodationId: Joi.number()
-            .required()
-            .messages({
-                'string.empty': res.__('Id of an accomodation cannot be empty'),
-                'any.required': res.__('Id of an accomodation is required')
-            }),
-        roomId: Joi.number()
-            .required()
-            .messages({
-                'string.empty': res.__('Id of a room cannot be empty'),
-                'any.required': res.__('Id of a room is required')
-            }),
-        checkInDate: Joi.string()
-            .required()
-            .messages({
-                'string.empty': res.__('Check-in date cannot be empty'),
-                'any.required': res.__('Check-in date is required')
-            }),
-        checkOutDate: Joi.string()
-            .required()
-            .messages({
-                'string.empty': res.__('Check-out date cannot be empty'),
-                'any.required': res.__('CHeck-out date is required')
-            }),
-    })
+  const schema = Joi.object({
+    accomodationId: Joi.number()
+      .required()
+      .messages({
+        'string.empty': res.__('Id of an accomodation cannot be empty'),
+        'any.required': res.__('Id of an accomodation is required')
+      }),
+    roomId: Joi.number()
+      .required()
+      .messages({
+        'string.empty': res.__('Id of a room cannot be empty'),
+        'any.required': res.__('Id of a room is required')
+      }),
+    checkInDate: Joi.string()
+      .required()
+      .messages({
+        'string.empty': res.__('Check-in date cannot be empty'),
+        'any.required': res.__('Check-in date is required')
+      }),
+    checkOutDate: Joi.string()
+      .required()
+      .messages({
+        'string.empty': res.__('Check-out date cannot be empty'),
+        'any.required': res.__('CHeck-out date is required')
+      }),
```

Comment:
can you validate dates as dates **joi.date()** instead of validating them as strings **joi.string()**